js/ad_manager_vast.js

```diff
@@ -2604,16 +2615,21 @@ OO.Ads.manager(function(_, $) {
      * @return {boolean} VPaid validated value
      */
     var _isValidVPaid = _.bind(function() {
-      var creative = currentAd.vpaidAd,
-          vpaidVersion = parseInt(creative.handshakeVersion('2.0')),
-          isValid = true;
+      var vpaidVersion = null;
+      try{
+        vpaidVersion = parseInt(currentAd.vpaidAd.handshakeVersion('2.0'));
```

Comment:
Also, do you want parseInt or parseFloat? since we could have minor updates to the standard later.